analyzers/tests/SonarAnalyzer.UnitTest/Helpers/PropertiesHelperTest.cs

```diff
@@ -51,15 +53,14 @@ public class PropertiesHelperTest
         public void ShouldAnalyzeGeneratedCode_WithNotBooleanValue_ReturnsFalse() =>
             GetSetting("ResourceTests\\NotBoolean\\SonarLint.xml").Should().BeFalse();
 
-        [TestMethod]
-        public void ShouldAnalyzeGeneratedCode_WithNotExistingPath_ReturnsFalse() =>
-            GetSetting("ResourceTests\\NotExistingFolder\\SonarLint.xml").Should().BeFalse();
-
         private static bool GetSetting(string filePath)
         {
             // Arrange
+            var content = SourceText.From(File.ReadAllText(filePath));
             var additionalText = new Mock<AdditionalText>();
-            additionalText.Setup(x => x.Path).Returns(filePath);
+            additionalText.Setup(x => x.Path).Returns("fakePath\\SonarLint.xml"); // use in-memory additional file
```

Comment:
please add a test when the path does not contain the "SonarLint.xml" _(clean as you code methodology, as the behavior should not have changed but we should have it documented in tests)_

Reply:
Done